modules/oe_theme_helper/oe_theme_helper.module

```diff
@@ -63,3 +63,14 @@ function oe_theme_helper_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter() for Accordion item.
+ */
+function oe_theme_helper_field_widget_oe_paragraphs_variants_form_alter(&$element, &$form_state, $context) {
+  if ($element['#paragraph_type'] !== 'oe_accordion_item') {
+    return;
+  }
+  // Hide icon field as it's not supported in ECL 2.
+  unset($element['subform']['field_oe_icon']);
```

Comment:
We should set the access to FALSE and also check if the field exists at all. It could be removed from the form display.